doc/es-Cluster.md

```diff
@@ -0,0 +1,93 @@
+﻿# Cluster

```

Comment:
No need to include files that have not changed.

Reply:
I don't know what happened here, I had translated this

Reply:
Do it